src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars__star {
+  display: flex;
+  place-content: center;
+  align-items: center;
+  width: 16px;
+  height: 16px;
+
+  &:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars {
+  display: flex;
+  align-content: center;
+  align-items: center;
+
+  &__star-image {
+    width: 13px;
+    height: 12px;
+    background-image: url('../images/star.svg');
+    background-size: cover;
+    background-position: center;
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) .stars__star-image,
```

Comment:
There is a mistake in the selector for `.stars--1 .stars__star:nth-child(-n + 1) .stars__star-image`. It looks like you're trying to select `.stars__star-image` within `.stars__star`, but `.stars__star-image` is already a modifier of `.stars__star`, so the space between `.stars__star` and `.stars__star-image` should not be there.